third_party/amd/lib/TritonAMDGPUTransforms/StreamPipelineV2.cpp

```diff
@@ -41,13 +62,52 @@ struct LoadInfo {
   bool usedByDot = false;
 };
 
+// Encapsulate stream pipelining
+// For each `scf.for` create a StreamPipeliner manager.
+class StreamPipeliner {
+public:
+  StreamPipeliner(scf::ForOp _forOp, int _numStages)
+      : forOp(_forOp), schedule(_numStages), numStages(_numStages),
+        axisInfoAnalysis(forOp->getParentOfType<ModuleOp>()) {
+    options.supportDynamicLoops = true;
+    options.peelEpilogue = true;
+    options.predicateFn = streamPredication;
+  }
+
+  void loadOpsToIndirectionLevelAndUse();
```

Comment:
Style: "computeLoadOpsTo.." to be consistent with others.

Reply:
I was using the original names, but since the interface changed I'll update the naming as well.